src/app/lodading.module.css

```
@@ -0,0 +1,75 @@
+.loading {
+  background-color: hsl(from var(--neutral-100) h s l / 0.5);
+  backdrop-filter: blur(3px);
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+
+  .loader {
+    display: block;
+    position: relative;
+    left: 50%;
+    top: 50%;
+    width: 6.3rem;
+    height: 6.3rem;
+    margin: -4.6rem 0 0 -4.5rem;
+    border-radius: 50%;
+    border: 3px solid transparent;
+    border-top-color: var(--primary-tint-60);
+    -webkit-animation: spin 2s linear infinite;
+    animation: spin 2s linear infinite;
+
+    .loader:before {
```

اینجا باید از & استفاده میکردید

```suggestion
    &::before {
```

---

درست کردم ممنون